docs/readthedocs/source/doc/Orca/Tutorial/k8s.md

```diff
@@ -217,6 +217,7 @@ git clone https://github.com/zalandoresearch/fashion-mnist.git
 mv /path/to/fashion-mnist/data/fashion/* /bigdl/nfsdata/dataset/FashionMNIST/raw
 
 # Extract FashionMNIST archives
+# May need to upgrade gzip before running the command
```

Comment:
gzip 1.10 or 1.12 can work, 1.5 fails with error
```
gzip: invalid option -- 'k'
Try `gzip --help' for more information.
```

Reply:
-k will keep the original file; without -k there will only be the extracted files